tests/test_travis_multiplePS/ps_test.cpp

```diff
@@ -0,0 +1,11 @@
+#include <Configuration.h>
```

Comment:
This file is the same or exactly the same as ps_test_mf. ps_test_mf2 seems redundant as well

Reply:
Fixed.